src/ripple/app/misc/impl/ValidatorSite.cpp

```diff
@@ -38,13 +39,21 @@ ValidatorSite::Site::Resource::Resource (std::string uri_)
     : uri {std::move(uri_)}
 {
     if (! parseUrl (pUrl, uri) ||
-        (pUrl.scheme != "http" && pUrl.scheme != "https"))
+        (pUrl.scheme != "http" && pUrl.scheme != "https"
+            && pUrl.scheme != "file") ||
```

Comment:
now that this supports `file:` protocol, we should consider whether to allow this from `Location` headers (redirects). I think there is no good use-case for redirects from `http/https:` to `file:` resources - in which case we should disallow. If we think there is a use-case, then I think we should just add a test case to verify the behavior.

Reply:
While I don't really see a problem in allowing such a redirect, my position is that it should _not_ be allowed.